tools/mull-cxx/mull-cxx.cpp

```diff
@@ -211,6 +212,12 @@ int main(int argc, char **argv) {
     filePathFilter->include(regex);
   }
 
+  if (tool::GitDiff.getValue().size() > 0) {
```

Comment:
I might have missed the right way of testing the presence of an optional CLI string argument. Do you know a better way?

Reply:
`hasValue()` maybe? Anywhow, it's recommended to use `empty()` instead of `size() == 0`

Reply:
`hasValue` returns true when there is no option provided. I have switched to `empty()`.